view.php

```diff
@@ -1,39 +1,50 @@
 <?php
+include 'common.php';
+
+$baseurl = 'http://infinity.unstable.life/Flashpoint/Data/Images';
+
+$keys =
+[
+	'id' => 'UUID',
+	'alternateTitles' => 'Alternate Titles',
+	'series' => 'Series',
+	'developer' => 'Developer',
+	'publisher' => 'Publisher',
+	'releaseDate' => 'Release Date',
+	'language' => 'Languages',
+	'applicationPath' => 'Application Path',
+	'launchCommand' => 'Launch Command'
+];
 
-$keys = array(
-    'id' => 'UUID',
-    'alternateTitles' => 'Alternate Titles',
-    'series' => 'Series',
-    'developer' => 'Developer',
-    'publisher' => 'Publisher',
-    'releaseDate' => 'Release Date',
-    'language' => 'Languages',
-    'applicationPath' => 'Application Path',
-    'launchCommand' => 'Launch Command'
-);
 $id = $_GET['id'];
-$blur = $_GET['blur'] == 'true';
 $db = new SQLite3('flashpoint.sqlite');
 $stmt = $db->prepare('SELECT * FROM game WHERE id = :id');
 $stmt->bindValue(':id', $id);
 $result = $stmt->execute();
 $game = $result->fetchArray(SQLITE3_ASSOC);
-if (!$game) {
-    header('HTTP/1.1 404 Not Found');
-    die('Game not found');
+$blur = is_extreme($game, $nsfw);
+
+if(!$game)
```

Comment:
Please keep the brace style

Reply:
They specifically throw the following errors in newer versions of PHP:
```
PHP Warning:  Use of undefined constant platform - assumed 'platform' (this will throw an Error in a future version of PHP) in FlashpointBrowser/search.php on line 36
PHP Warning:  Use of undefined constant title - assumed 'title' (this will throw an Error in a future version of PHP) in FlashpointBrowser/search.php on line 36
```
But I added it back in if that's the preference.

Reply:
I'm referring to the if statement:
```php
if($game)
{
}
```
vs
```php
if ($game) {
}
```
What you're referring to is a separate issue in search.php